include/picongpu/plugins/particleMerging/ParticleMerger.kernel

```diff
@@ -116,10 +116,11 @@ namespace picongpu
                     const uint32_t workerIdx
                         = DataSpaceOperations<simDim>::template map<SuperCellSize>(cellIdx % SuperCellSize::toRT());
 
+                    auto const superCellIdx = pmacc::DataSpace<simDim>{cellIdx / SuperCellSize::toRT()};
                     auto forEachParticle = pmacc::particles::algorithm::acc::makeForEach<numWorkers>(
                         workerIdx,
                         particlesBox,
-                        cellIdx / static_cast<pmacc::math::Int<simDim>>(SuperCellSize::toRT()));
+                        superCellIdx);
```

Comment:
I am not sure why the type cast of `SuperCellSize::toRT()` was there, but at least now it seems to work without the cast as well, so removed it.

Reply:
The cast was in because CUSTL assumes `math::Vector` but looks like it is working without an cast too.